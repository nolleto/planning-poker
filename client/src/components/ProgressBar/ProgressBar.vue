<template>
  <div :class="bootstrap.progress">
    <div
      :class="[
        $style.progressBar,
        bootstrap.progressBar,
        bootstrap.progressBarStriped,
        bootstrap.progressBarAnimated
      ]"
      role="progressbar"
      aria-valuenow="100"
      aria-valuemin="0"
      aria-valuemax="100"
    >
      <slot></slot>
    </div>
  </div>
</template>

<script>
import styles from '@/styles'

export default {
  name: 'ProgressBar',

  computed: {
    bootstrap: () => styles.bootstrap
  }
}
</script>

<style module>
  .progressBar{
    width: 100%;
  }
</style>
