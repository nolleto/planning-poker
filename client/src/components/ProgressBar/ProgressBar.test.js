import ProgressBar from './index'
import { shallowMount } from '@vue/test-utils'

const shallowComponent = () => {
  return shallowMount(ProgressBar)
}

describe('ProgressBar component', () => {
  let wrapper

  describe('Snapshots', () => {
    beforeAll(() => {
      wrapper = shallowComponent()
    })

    it('matches', () => {
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
