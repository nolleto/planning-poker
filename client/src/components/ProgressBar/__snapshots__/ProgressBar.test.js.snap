// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar component Snapshots matches 1`] = `
<div
  class="progress"
>
  <div
    aria-valuemax="100"
    aria-valuemin="0"
    aria-valuenow="100"
    class="progressBar progressBar progressBarStriped progressBarAnimated"
    role="progressbar"
  />
</div>
`;
